### PR TITLE
mo_unpack test skip

### PR DESCRIPTION
**Pull request by marqh**
added mo_unpack test skipping

### CODE REVIEW

**Comment by rhattersley** [comment]
Surely we don't need to skip *all* of these tests if mo_pack is not installed!

**Comment by marqh** [comment]
> Surely we don't need to skip all of these tests if mo_pack is not installed!

apologies, I misunderstood the failure modes.  This is indeed limited to a small number of tests, reapplied to affect these ones only

**Comment by marqh** [comment]
@rhattersley is this PR now suitable to merge?
thank you

**Comment by pelson** [comment]
How we maintain this going forwards, I do not know, but it looks good for now.

File: lib/iris/tests/integration/test_FieldsFileVariant.py [hunk 1]
```diff
@@ -24,12 +24,22 @@
...
 import tempfile
+import unittest
 
 import numpy as np
 
 from iris.experimental.um import (Field, Field2, Field3, FieldsFileVariant,
                                   FixedLengthHeader)
 
+try:
+    import mo_unpack
```
Comment (ajdawson) [inline lib/iris/tests/integration/test_FieldsFileVariant.py]: What is `mo_unpack`? Should this be `mo_pack` instead? I'm getting a bunch of skipped tests saying I need `mo_unpack`...

Comment (ajdawson) [inline lib/iris/tests/integration/test_FieldsFileVariant.py]: I'm thinking this is wrong, since this causes tests to be skipped in the Travis-CI build...

**Pull request merged**